vars/sectionBuildAndTest.groovy

```diff
@@ -98,6 +98,7 @@ def call(params) {
                         # in case anyone doesn't have a trailing new line in their file
                         echo -e '\n' >> .dockerignore
                         cat .dockerignore_build >> .dockerignore
+                        sed -i 's/-e//g' .dockerignore
```

Comment:
what's this for?

Reply:
For some reason the docker ignore is appending `-e` to the `.jar` file. I will change it to replace `.jar-e` with `.jar` so it doesn't muck up images with `-e` in their names. I think it's something to do with the echo

Reply:
probably needs a bit more debugging, seems quite odd =/